pd-LukaszPuszkarewicz/pracaDomowa1.js

```diff
@@ -40,14 +63,28 @@ const people = [
 */
 
 //  DANE WEJŚCIOWE
-const people = [
+const people2 = [
   {
     firstName: "Bartolomeo",
     lastName: "Lozano",
     nickname: "Rabona",
+    // Metoda introduceYourself jest właściwością pierwszego obiektu w tablicy people2. Gdy jest wywoływana loguje w konsoli wiadomość skonstruowaną ze właściwości obiektu oraz stringa zapisanego za pomocą template literals oraz słówka this które jest uzwane aby uzyskać dostęp do właściwości obiektu.
+    introduceYourself: function () {
```

Comment:
Proponuję zadeklarować tą funkcję poza obiektem, pozwoli to na jej użycie w innym miejscu.